### PR TITLE
Prepare version 0.24

### PR DESCRIPTION
**Pull request by mortenpi**
Final two changes:

* ~Add a note about the GitHub Actions feature not completely working yet and the API being experimental for now (#1177).~ See #1183
* Make the headings clickable again, to make it easier to get links to in-page headings.
* Fix the SVG circle on showcase page

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,8 +24,6 @@
 
 * ![Feature][badge-feature] Deployment is now more customizable and thus not as tied to Travis CI as before. ([#1147][github-1147], [#1171][github-1171])
```
Comment (fredrikekre) [inline CHANGELOG.md]: Mark this as experimental too?

Comment (mortenpi) [reply CHANGELOG.md]: I'd say that it's only the GitHub Actions part that needs to be experimental. The rest of the refactoring is still good even if we'd nuke the Actions part.

Comment (fredrikekre) [reply CHANGELOG.md]: Right, but I wanted to redo that interface to properly support PR previews :p

Comment (fredrikekre) [reply CHANGELOG.md]: Actually, I have that almost finished on some branch. Let me push that and we can see.

Comment (fredrikekre) [reply CHANGELOG.md]: #1180

**Comment by fredrikekre** [comment]
See also #1183

**Pull request merged**